timesketch/lib/charts/barchart.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Barcharts."""
+
+from __future__ import unicode_literals
+
+import altair as alt
+
+from timesketch.lib.charts import manager
+from timesketch.lib.charts import interface
+
+
+class BarChart(interface.BaseChart):
+    """Barchart object."""
+
+    NAME = 'barchart'
+
+    def __init__(self, data):
+        """Initialize the chart object.
+
+        Args:
+            data: Aggregation result from an aggregator object as a dict.
+        """
+        super(BarChart, self).__init__(data)
```

Comment:
no need to keep the __init__ here, you are not overwriting it at all

Reply:
I'm initializing my super class with "data" here.

Reply:
same as before, you inherit the class if you don't overwrite it, so it's not needed